_data/langs/de.yml

```diff
@@ -187,26 +187,26 @@ roadmap:
   i2P_android_wallet:
     header: "I2P Android Geldbörse"
     paragraph: "Android Geldbörse ausschließlich nur für anonyme Transaktionen (kurz vor Fertigstellung)"
-    date: "Q4, 2017"
+    date: "Q1, 2018"
   core_wallet_release:
     header: "Core Wallet 3.0 Release Stage 1"
     paragraph: "Wallet UI Overhaul, VISP, Bloom Filters, Atomic Swaps Capability"
     date: "<font color='#08b4e0'>Veröffentlicht</font> 15. November 2017"
   core_wallet_release_2:
     header: "Core Wallet 3.0 Release Stage 2"
```

Comment:
I think this should be "Core Wallet 4.0 Release Stage 1"

Reply:
added